package.json

```diff
@@ -79,7 +80,7 @@
     "react-autosuggest": "^9.4.2",
     "react-datepicker": "^2.0.0",
     "react-dev-utils": "^7.0.0",
-    "react-dom": "^16.8.1",
+    "react-dom": "npm:@hot-loader/react-dom",
```

Comment:
Enables newest react features to be hot reloaded, and gets rid of the console warning.